scss/atoms/buttons/_mixins.scss

```diff
@@ -0,0 +1,64 @@
+/*
+* ==========================================================================
+* Buttons (Mixins)
+* ==========================================================================
+*/
+
+@mixin ui-button($color, $bgcolor, $disabled: false) {
+	background: $bgcolor;
+	border: 0;
+	color: $color;
+	display: inline-block;
+	font-weight: 500;
+	margin: .25em;
+	text-align: center;
+	text-decoration: none;
+	text-transform: uppercase;
```

Comment:
Once the jsass PR lands I will probably make that line into one of the configurable variables that the theme roller can set, for those who prefer not to have uppercase buttons.

Sent from my iPhone

> On Jul 17, 2015, at 1:48 AM, Chris Rebert <notifications@github.com> wrote:
> 
> In scss/atoms/buttons/_mixins.scss:
> 
> > +/*
> > +* ==========================================================================
> > +* Buttons (Mixins)
> > +* ==========================================================================
> > +*/
> > +
> > +@mixin ui-button($color, $bgcolor, $disabled: false) {
> > +	background: $bgcolor;
> > +	border: 0;
> > +	color: $color;
> > +	display: inline-block;
> > +	font-weight: 500;
> > +	margin: .25em;
> > +	text-align: center;
> > +	text-decoration: none;
> > +	text-transform: uppercase;
> Too opinionated, imho
> 
> —
> Reply to this email directly or view it on GitHub.
>